### PR TITLE
Fix shadow scaling

### PR DESCRIPTION
**Pull request by shamansir**
@skavish @mcm69 Please review

### CODE REVIEW

File: src/anm/player.js [hunk 1]
```diff
@@ -916,6 +921,48 @@ Player.prototype.size = function(width, height) {
 };
 
 /**
+ * @method factor
+ *
+ * Returns the difference factor between player size and animation size,
+ * using fit by largest side. _Does not_ counts scene zoom, since it does not
```
Comment (mcm69) [inline src/anm/player.js]: does not _count_

File: src/anm/render.js [hunk 2]
```diff
@@ -89,8 +89,8 @@ function r_at(time, dt, ctx, anim, width, height, zoom, rib_color, before, after
         } finally { ctx.restore(); }
     } else {
         r_with_ribbons(ctx, width, height,
-                              anim.width, anim.height,
-                              rib_color,
+                            anim.width, anim.height,
+                            rib_color,
```
Comment (mcm69) [inline src/anm/render.js]: why not kill the linebreaks here altogether?

**Comment by mcm69** [comment]
looks good

**Comment by skavish** [comment]
lgtm

File: src/anm/render.js (hunk 2 above)
Comment (shamansir) [inline src/anm/render.js]: I like grouping things that visually group together, it reads better and feels better than having long lines of arguments.

File: src/anm/player.js (hunk 1 above)
Comment (shamansir) [inline src/anm/player.js]: Will fix, thanks!

**Pull request merged**